diff -ur dee-1.0.6.orig/src/dee-filter.h /dee-1.0.6/src/dee-filter.h
--- src/dee-filter.h	2012-02-25 00:46:45.000000000 +0800
+++ src/dee-filter.h	2012-03-18 00:37:42.255488846 +0800
@@ -30,8 +30,6 @@
 
 G_BEGIN_DECLS
 
-typedef struct _DeeFilter DeeFilter;
-
 /**
  * DeeFilterMapFunc:
  * @orig_model: The model containing the original data to filter
diff -ur dee-1.0.6.orig/src/dee-model-reader.h dee-1.0.6/src/dee-model-reader.h
--- src/dee-model-reader.h	2011-12-19 22:11:45.000000000 +0800
+++ src/dee-model-reader.h	2012-03-18 01:04:06.087479112 +0800
@@ -30,8 +30,6 @@
 
 G_BEGIN_DECLS
 
-typedef struct _DeeFilter DeeFilter;
-
 /**
  * DeeModelReaderFunc:
  * @model: The model being indexed
