diff -ur virtuoso-opensource-6.1.5.orig/binsrc/bpel/make_vad.sh virtuoso-opensource-6.1.5/binsrc/bpel/make_vad.sh
--- binsrc/bpel/make_vad.sh	2012-03-09 00:16:31.000000000 +0800
+++ binsrc/bpel/make_vad.sh	2012-03-19 09:04:53.830490303 +0800
@@ -42,7 +42,7 @@
 THOST=${THOST-localhost}
 TPORT=${TPORT-8445}
 PORT=${PORT-1940}
-ISQL=${ISQL-isql}
+ISQL=${ISQL-isql-v}
 VAD_NAME="bpel4ws"
 VAD_PKG_NAME="bpel"
 VAD_NAME_DEVEL="$VAD_PKG_NAME"_filesystem.vad
@@ -665,6 +665,7 @@
     LOG "***ABORTED: BPEL4WS PACKAGING, isql is not available"
     exit 1
 fi
+SERVER=../../binsrc/virtuoso/virtuoso
 $SERVER -? 2>/dev/null 1>/dev/null 
 if [ $? -eq 127 ] ; then
     LOG "***ABORTED: BPEL4WS PACKAGING, server is not available"
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/rdf_mappers/make_vad.sh virtuoso-opensource-6.1.5/binsrc/rdf_mappers/make_vad.sh
--- binsrc/rdf_mappers/make_vad.sh	2012-03-14 21:02:05.000000000 +0800
+++ binsrc/rdf_mappers/make_vad.sh	2012-03-19 08:59:45.062506344 +0800
@@ -41,7 +41,7 @@
 THOST=${THOST-localhost}
 TPORT=${TPORT-8445}
 PORT=${PORT-1940}
-ISQL=${ISQL-isql}
+ISQL=${ISQL-isql-v}
 VAD_NAME="rdf_mappers"
 VAD_PKG_NAME="rdf_mappers"
 VAD_DESC="RDF Mappers"
@@ -498,7 +498,8 @@
     LOG "***ABORTED: $VAD_DESC PACKAGING, isql is not available"
     exit 1
 fi
-$SERVER -? 2>/dev/null 1>/dev/null 
+SERVER=../../binsrc/virtuoso/virtuoso
+$SERVER -?  2>/dev/null 1>/dev/null
 if [ $? -eq 127 ] ; then
     LOG "***ABORTED: $VAD_DESC PACKAGING, server is not available"
     exit 1
diff -ur virtuoso-opensource-6.1.5/binsrc/b3s/make_vad.sh virtuoso-opensource-6.1.5/binsrc/b3s/make_vad.sh
--- binsrc/b3s/make_vad.sh	2012-03-09 00:16:30.000000000 +0800
+++ binsrc/b3s/make_vad.sh	2012-03-19 10:19:54.999265886 +0800
@@ -191,7 +191,7 @@

   # components which are not in their place
   cp -Rf $HOME/binsrc/oat vad/vsp/fct
-  cp -Rf $HOME/binsrc/samples/dbpedia/vsp/statics vad/vsp/fct/rdfdesc
+  cp -Rf $HOME/binsrc/dbpedia/vsp/statics vad/vsp/fct/rdfdesc
   cp -Rf $HOME/binsrc/rdf_mappers/virt_rdf_label.sql vad/vsp/fct
   cp -Rf styles vad/vsp/fct/rdfdesc
   cp -Rf s vad/vsp/fct/rdfdesc
@@ -475,6 +475,7 @@
     LOG "***ABORTED: $VAD_DESC PACKAGING, isql is not available"
     exit 1
 fi
+SERVER=../../binsrc/virtuoso/virtuoso
 $SERVER -? 2>/dev/null 1>/dev/null
 if [ $? -eq 127 ] ; then
     LOG "***ABORTED: $VAD_DESC PACKAGING, server is not available"

@@ -490,10 +490,10 @@
 sticker_init 0
 virtuoso_init
 #vad_create $STICKER_FS $VAD_NAME_DEVEL
-vad_create $STICKER_DAV $VAD_NAME_RELEASE
+#vad_create $STICKER_DAV $VAD_NAME_RELEASE
 virtuoso_shutdown
 #chmod 644 $VAD_NAME_DEVEL
-chmod 644 $VAD_NAME_RELEASE
+#chmod 644 $VAD_NAME_RELEASE
 directory_clean

 CHECK_LOG
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/tutorial/make_vad.sh virtuoso-opensource-6.1.5/binsrc/tutorial/make_vad.sh
--- binsrc/tutorial/make_vad.sh	2012-03-09 00:16:58.000000000 +0800
+++ binsrc/tutorial/make_vad.sh	2012-03-19 12:29:50.818876618 +0800
@@ -18,7 +18,7 @@
 THOST=${THOST-localhost}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
-ISQL=${ISQL-isql}
+ISQL=${ISQL-isql-v}
 DSN="$HOST:$PORT"
 HOST_OS=`uname -s | grep WIN`
 NEED_VERSION=04.00.2803
@@ -604,6 +604,7 @@
     LOG "***ABORTED: tutorial PACKAGING, isql is not available"
     exit 1
 fi
+SERVER=../../binsrc/virtuoso/virtuoso
 $SERVER -? 2>/dev/null 1>/dev/null 
 if [ $? -eq 127 ] ; then
     LOG "***ABORTED: tutorial PACKAGING, server is not available"
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/yacutia/mkvad.sh virtuoso-opensource-6.1.5/binsrc/yacutia/mkvad.sh
--- binsrc/yacutia/mkvad.sh	2012-03-09 00:17:11.000000000 +0800
+++ binsrc/yacutia/mkvad.sh	2012-03-19 13:21:10.314725783 +0800
@@ -384,6 +383,7 @@
     LOG "***ABORTED: CONDUCTOR PACKAGING, isql is not available"
     exit 1
 fi
+SERVER=../../binsrc/virtuoso/virtuoso
 $SERVER -? 2>/dev/null 1>/dev/null
 if [ $? -eq 127 ] ; then
     LOG "***ABORTED: CONDUCTOR PACKAGING, server is not available"
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/samples/demo/mkdoc.sh virtuoso-opensource-6.1.5/binsrc/samples/demo/mkdoc.sh
--- binsrc/samples/demo/mkdoc.sh	2012-03-09 00:16:43.000000000 +0800
+++ binsrc/samples/demo/mkdoc.sh	2012-03-19 23:29:50.494805009 +0800
@@ -316,7 +316,7 @@
   mkdir vad
   mkdir vad/data
   mkdir vad/data/doc
-#  mkdir vad/data/doc/html
+  mkdir vad/data/doc/html
   mkdir vad/data/doc/code
   mkdir vad/data/doc/images
   mkdir vad/data/doc/images/inst
@@ -358,10 +358,6 @@

   $CP docsrc/images/tree/*.gif vad/data/doc/images/tree/.

-  $CP docsrc/images/ui/*.jpg vad/data/doc/images/ui/.
-  $CP docsrc/images/ui/*.gif vad/data/doc/images/ui/.
-  $CP docsrc/images/ui/*.png vad/data/doc/images/ui/.
-
   cp mksearch.sql vad/data/doc/code/.
   cp drop.sql vad/data/doc/code/.
   cp doc_sql_rdf.sql vad/data/doc/code/.
@@ -500,6 +500,7 @@
     LOG "***ABORTED: DOCUMENTATION PACKAGING, isql is not available"
     exit 1
 fi
+SERVER="${S}/binsrc/virtuoso/virtuoso-t"
 $SERVER -? 2>/dev/null 1>/dev/null 
 if [ $? -eq 127 ] ; then
     LOG "***ABORTED: DOCUMENTATION PACKAGING, server is not available"
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/dbpedia/make_vad.sh virtuoso-opensource-6.1.5/binsrc/dbpedia/make_vad.sh
--- binsrc/dbpedia/make_vad.sh	2012-03-02 20:43:27.000000000 +0800
+++ binsrc/dbpedia/make_vad.sh	2012-03-21 00:34:51.560528366 +0800
@@ -35,7 +35,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_fs.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8445}
 PORT=${PORT-1940}
@@ -422,6 +422,7 @@
     LOG "***ABORTED: $VAD_DESC PACKAGING, isql is not available"
     exit 1
 fi
+SERVER="${S}/binsrc/virtuoso/virtuoso-t"
 $SERVER -? 2>/dev/null 1>/dev/null 
 if [ $? -eq 127 ] ; then
     LOG "***ABORTED: $VAD_DESC PACKAGING, server is not available"
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/sync/make_vad.sh virtuoso-opensource-6.1.5/binsrc/sync/make_vad.sh
--- binsrc/sync/make_vad.sh	2012-03-09 00:16:46.000000000 +0800
+++ binsrc/sync/make_vad.sh	2012-03-21 00:32:32.512528982 +0800
@@ -34,7 +34,7 @@
 LOGFILE="${LOGDIR}/make_syncml_vad.log"
 STICKER="make_syncml_vad.xml"
 STICKER_NAME="make_syncml_vad.xml"
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
@@ -108,7 +108,7 @@
   $myrm -f *.lck
   if [ "x$HOST_OS" != "x" ]
   then
-      $BUILD/../bin/virtuoso-odbc-t +foreground &
+      ${S}/binsrc/virtuoso/virtuoso-t +foreground &
   else
       virtuoso +wait
   fi
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/vsp/admin/debug/make_vad.sh virtuoso-opensource-6.1.5/binsrc/vsp/admin/debug/make_vad.sh
--- binsrc/vsp/admin/debug/make_vad.sh	2012-03-09 00:17:08.000000000 +0800
+++ binsrc/vsp/admin/debug/make_vad.sh	2012-03-21 00:31:38.872527229 +0800
@@ -33,7 +33,7 @@
 VERSION="1.00.02"
 LOGFILE="${LOGDIR}/make_http_debug_vad.log"
 STICKER="make_http_debug_vad.xml"
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
@@ -74,7 +74,7 @@
   $myrm -f *.lck
   if [ "x$HOST_OS" != "x" ]
 	then
-	  $BUILD/../bin/virtuoso-odbc-t +foreground &
+	  ${S}/binsrc/virtuoso/virtuoso-t +foreground &
   else
 	  virtuoso +wait
   fi
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/samples/demo/make_vad.sh virtuoso-opensource-6.1.5/binsrc/samples/demo/make_vad.sh
--- binsrc/samples/demo/make_vad.sh	2012-03-09 00:16:42.000000000 +0800
+++ binsrc/samples/demo/make_vad.sh	2012-03-21 02:52:20.624118244 +0800
@@ -263,14 +263,13 @@
   cp -f EMP9                                                    vad/data/demo/sql
   cp -f thalia_test/* 											vad/data/demo/Thalia
   cp -f $HOME/docsrc/xmlsource/*                                vad/data/demo/docsrc
-  cp -f $HOME/docsrc/xmlsource/DocBook/*                        vad/data/demo/docsrc/DocBook
-  cp -f $HOME/docsrc/xmlsource/DocBook/ent/*                    vad/data/demo/docsrc/DocBook/ent
-  cp -f $HOME/docsrc/xmlsource/funcref/*                        vad/data/demo/docsrc/funcref
-  cp -f $HOME/docsrc/xmlsource/funcref/*                        vad/data/demo/docsrc/funcref
-  cp -f $HOME/docsrc/images/*                                   vad/data/demo/images
-  cp -f $HOME/docsrc/images/tree/*                              vad/data/demo/images/misc
-  cp -f $HOME/docsrc/images/misc/*                              vad/data/demo/images/tree
-  cp -f $HOME/docsrc/stylesheets/*                              vad/data/demo/stylesheets
+  cp -a $HOME/docsrc/xmlsource/DocBook/                         vad/data/demo/docsrc/
+  cp -a $HOME/docsrc/xmlsource/funcref/                         vad/data/demo/docsrc/
+  cp -a $HOME/docsrc/xmlsource/bpel                             vad/data/demo/docsrc/  
+  cp -a $HOME/docsrc/xmlsource/vspx_ref                         vad/data/demo/docsrc/
+  cp -a $HOME/docsrc/xmlsource/xmlref 				 vad/data/demo/docsrc/
+  cp -a $HOME/docsrc/images/                                    vad/data/demo/
+  cp -f $HOME/docsrc/stylesheets/*                              vad/data/demo/
   cp -f $HOME/docsrc/stylesheets/sections/doc.css               vad/data/demo/docsrc
   cp -f $HOME/docsrc/stylesheets/sections/openlink.css          vad/data/demo/docsrc
   cp -f $HOME/docsrc/releasenotes/*                             vad/data/demo/releasenotes
@@ -573,6 +571,7 @@
   do_command_safe $DSN "commit work"
   do_command_safe $DSN "checkpoint"
 }
+SERVER="${S}/binsrc/virtuoso/virtuoso-t"
 
 BANNER "STARTED PACKAGING DEMO VAD"
 STOP_SERVER
diff -ur virtuoso-opensource-6.1.5.orig/binsrc/samples/sparql_demo/make_vad.sh 
virtuoso-opensource-6.1.5/binsrc/samples/sparql_demo/make_vad.sh
--- binsrc/samples/sparql_demo/make_vad.sh	2012-03-09 00:16:43.000000000 +0800
+++ binsrc/samples/sparql_demo/make_vad.sh	2012-03-21 03:13:11.120052314 +0800
@@ -35,7 +35,7 @@
 LOGFILE="${LOGDIR}/make_sparql_demo_vad.log"
 STICKER="make_sparql_demo_vad.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
@@ -397,6 +397,7 @@
     do_command_safe $DSN "commit work"
     do_command_safe $DSN "checkpoint"
 }
+SERVER="${S}/binsrc/virtuoso/virtuoso-t"
 
 BANNER "STARTED PACKAGING SPARQL_DEMO VAD"
 STOP_SERVER
diff -ur virtuoso-odbc-6.1.5.orig/binsrc/isparql/make_vad.sh virtuoso-opensource-6.1.5/binsrc/isparql/make_vad.sh
--- binsrc/isparql/make_vad.sh	2012-03-09 00:16:40.000000000 +0800
+++ binsrc/isparql/make_vad.sh	2012-03-21 03:45:30.864970744 +0800
@@ -34,7 +34,7 @@
 LOGFILE="${LOGDIR}/make_isparql_vad.log"
 STICKER="make_isparql_vad.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
@@ -417,6 +417,7 @@
   do_command_safe $DSN "commit work"
   do_command_safe $DSN "checkpoint"
 }
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 BANNER "STARTED PACKAGING iSPARQL VAD"
 STOP_SERVER

diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Framework/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Framework/make_vad.sh
--- appsrc/ODS-Framework/make_vad.sh	2012-03-09 00:16:24.000000000 +0800
+++ appsrc/ODS-Framework/make_vad.sh	2012-03-21 08:50:45.600045208 +0800
@@ -34,7 +34,7 @@
 LOGFILE="${LOGDIR}/make_ods_vad.log"
 STICKER="make_ods_vad.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1970}
 PORT=`expr $PORT + 10`
@@ -561,6 +561,7 @@
   do_command_safe $DSN "commit work"
   do_command_safe $DSN "checkpoint"
 }
+SERVER="${S}/binsrc/virtuoso/virtuoso"
 
 STOP_SERVER
 $myrm $LOGFILE 2>/dev/null
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Framework/oauth/make_vad.sh 
virtuoso-opensource-6.1.5/appsrc/ODS-Framework/oauth/make_vad.sh
--- appsrc/ODS-Framework/oauth/make_vad.sh	2012-03-09 00:16:27.000000000 +0800
+++ appsrc/ODS-Framework/oauth/make_vad.sh	2012-03-21 08:34:51.017095198 +0800
@@ -35,7 +35,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_fs.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8445}
 PORT=${PORT-1940}
@@ -411,6 +411,7 @@
     LOG "***ABORTED: $VAD_DESC PACKAGING, isql is not available"
     exit 1
 fi
+SERVER="${S}/binsrc/virtuoso/virtuoso-t"
 $SERVER -? 2>/dev/null 1>/dev/null 
 if [ $? -eq 127 ] ; then
     LOG "***ABORTED: $VAD_DESC PACKAGING, server is not available"
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Addressbook/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Addressbook/make_vad.sh
--- appsrc/ODS-Addressbook/make_vad.sh	2012-03-09 00:16:19.000000000 +0800
+++ appsrc/ODS-Addressbook/make_vad.sh	2012-03-21 18:55:28.024822076 +0800
@@ -35,7 +35,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_filesystem.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1940}
 TPORT=${TPORT-`expr $PORT + 1000`}
@@ -425,6 +425,7 @@
 echo '----------------------'
 echo 'AddressBook Manager Application VAD create'
 echo '----------------------'
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Blog/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Blog/make_vad.sh
--- appsrc/ODS-Blog/make_vad.sh	2012-03-09 00:16:19.000000000 +0800
+++ appsrc/ODS-Blog/make_vad.sh	2012-03-21 18:56:24.296819819 +0800
@@ -34,7 +34,7 @@
 STICKER_NAME="make_ods_blog_vad.xml"
 STICKER="${LOGDIR}/$STICKER_NAME"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
@@ -458,6 +458,7 @@
   do_command_safe $DSN "commit work"
   do_command_safe $DSN "checkpoint"
 }
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 BANNER "STARTED PACKAGING BLOG VAD"
 STOP_SERVER
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Bookmark/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Bookmark/make_vad.sh
--- appsrc/ODS-Bookmark/make_vad.sh	2012-03-09 00:16:20.000000000 +0800
+++ appsrc/ODS-Bookmark/make_vad.sh	2012-03-21 18:57:04.624818323 +0800
@@ -35,7 +35,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_filesystem.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1940}
 TPORT=${TPORT-`expr $PORT + 1000`}
@@ -423,6 +423,7 @@
 echo '----------------------'
 echo 'Bookmark Manager Application VAD create'
 echo '----------------------'
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Briefcase/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Briefcase/make_vad.sh
--- appsrc/ODS-Briefcase/make_vad.sh	2012-03-09 00:16:21.000000000 +0800
+++ appsrc/ODS-Briefcase/make_vad.sh	2012-03-21 18:57:41.024817118 +0800
@@ -35,7 +35,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_filesystem.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1940}
 TPORT=${TPORT-`expr $PORT + 1000`}
@@ -440,6 +440,7 @@
 echo '----------------------'
 echo 'Briefcase Application VAD create'
 echo '----------------------'
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Calendar/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Calendar/make_vad.sh
--- appsrc/ODS-Calendar/make_vad.sh	2012-03-09 00:16:22.000000000 +0800
+++ appsrc/ODS-Calendar/make_vad.sh	2012-03-21 18:58:18.520813776 +0800
@@ -33,7 +33,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_filesystem.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1940}
 TPORT=${TPORT-`expr $PORT + 1000`}
@@ -425,6 +425,7 @@
 echo '----------------------'
 echo 'Calendar Manager Application VAD create'
 echo '----------------------'
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Community/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Community/make_vad.sh
--- appsrc/ODS-Community/make_vad.sh	2012-03-09 00:16:22.000000000 +0800
+++ appsrc/ODS-Community/make_vad.sh	2012-03-21 18:58:47.263814503 +0800
@@ -35,7 +35,7 @@
 STICKER="make_vad.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
 SERVER=${SERVER-virtuoso}
-THOST=${THOST-localhost}
+THOST=${THOST-localhost-t}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
 ISQL=${ISQL-isql}
@@ -405,6 +405,7 @@
 echo '------------------------'
 echo 'Community VAD create'
 echo '------------------------'
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Discussion/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Discussion/make_vad.sh
--- appsrc/ODS-Discussion/make_vad.sh	2012-03-09 00:16:22.000000000 +0800
+++ appsrc/ODS-Discussion/make_vad.sh	2012-03-21 18:59:41.232812775 +0800
@@ -33,7 +33,7 @@
 LOGDIR=`pwd`
 LOGFILE="${LOGDIR}/make_nntpf_vad.log"
 STICKER="make_nntpf_vad.xml"
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
@@ -404,6 +404,7 @@
   do_command_safe $DSN "commit work"
   do_command_safe $DSN "checkpoint"
 }
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-FeedManager/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-FeedManager/make_vad.sh
--- appsrc/ODS-FeedManager/make_vad.sh	2012-03-09 00:16:23.000000000 +0800
+++ appsrc/ODS-FeedManager/make_vad.sh	2012-03-21 19:00:17.504809977 +0800
@@ -35,7 +35,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_filesystem.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1940}
 TPORT=${TPORT-`expr $PORT + 1000`}
@@ -440,6 +440,7 @@
 echo '----------------------'
 echo 'Feed Manager Application VAD create'
 echo '----------------------'
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Gallery/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Gallery/make_vad.sh
--- appsrc/ODS-Gallery/make_vad.sh	2012-03-09 00:16:27.000000000 +0800
+++ appsrc/ODS-Gallery/make_vad.sh	2012-03-21 19:00:44.799806932 +0800
@@ -33,7 +33,7 @@
 LOGFILE="${LOGDIR}/make_vad.log"
 STICKER="make_vad.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1940}
 TPORT=${TPORT-`expr $PORT + 1000`}
@@ -423,6 +423,7 @@
 echo '----------------------'
 echo 'WEB Gallery VAD create'
 echo '----------------------'
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Polls/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Polls/make_vad.sh
--- appsrc/ODS-Polls/make_vad.sh	2012-03-09 00:16:28.000000000 +0800
+++ appsrc/ODS-Polls/make_vad.sh	2012-03-21 19:01:02.704807067 +0800
@@ -33,7 +33,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_filesystem.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1940}
 TPORT=${TPORT-`expr $PORT + 1000`}
@@ -397,6 +397,7 @@
 echo '----------------------'
 echo 'Polls Manager Application VAD create'
 echo '----------------------'
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"

 STOP_SERVER
 directory_clean
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-WebMail/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-WebMail/make_vad.sh
--- appsrc/ODS-WebMail/make_vad.sh	2012-03-09 00:16:28.000000000 +0800
+++ appsrc/ODS-WebMail/make_vad.sh	2012-03-21 19:01:35.264804729 +0800
@@ -35,7 +35,7 @@
 STICKER_DAV="vad_dav.xml"
 STICKER_FS="vad_filesystem.xml"
 PACKDATE=`date +"%Y-%m-%d %H:%M"`
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 PORT=${PORT-1940}
 TPORT=${TPORT-`expr $PORT + 1000`}
@@ -427,7 +427,7 @@
 echo '----------------------'
 echo 'oMail Application VAD create'
 echo '----------------------'
-
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 STOP_SERVER
 directory_clean
 version_init
diff -ur virtuoso-opensource-6.1.5.orig/appsrc/ODS-Wiki/make_vad.sh virtuoso-opensource-6.1.5/appsrc/ODS-Wiki/make_vad.sh
--- appsrc/ODS-Wiki/make_vad.sh	2012-03-09 00:16:29.000000000 +0800
+++ appsrc/ODS-Wiki/make_vad.sh	2012-03-21 19:01:59.520803896 +0800
@@ -33,7 +33,7 @@
 LOGFILE="${LOGDIR}/make_wiki_vad.log"
 STICKER=`pwd`"/make_wiki_vad.xml"
 STICKER_NAME="make_wiki_vad.xml"
-SERVER=${SERVER-virtuoso}
+SERVER=${SERVER-virtuoso-t}
 THOST=${THOST-localhost}
 TPORT=${TPORT-8440}
 PORT=${PORT-1940}
@@ -502,6 +502,7 @@
   LOG "VAD uninstallation check..."
   do_command_safe $DSN "VAD_UNINSTALL('wiki/$VERSION');"    
 }
+SERVER="$HOME/binsrc/virtuoso/virtuoso-t"
 
 virtuoso_shutdown
 directory_clean


